sklearn/covariance/empirical_covariance_.py

```diff
@@ -79,6 +80,8 @@ def empirical_covariance(X, assume_centered=False):
     else:
         covariance = np.cov(X.T, bias=1)
 
+    if covariance.ndim == 0:
```

Comment:
Not behaving as the doc says causes problems because `check_array` does not convert the value returned to a 2D array. Should I let it be here or file a separate PR for this ?